package.json

```diff
@@ -675,6 +675,13 @@
           "type": "number",
           "description": "Threshold for reporting number of lines changed.",
           "default": 2
+        },
+        "vim.customDigraphs": {
+          "type": "object",
+          "description": "Custom digraph shortcuts for inserting special characters, expressed as UTF16 code points",
+          "default": {
+            "R!": ["🚀", [55357, 56960]]
```

Comment:
Did you mean to leave this default digraph here? What shall happen if a user puts a custom digraph here which is already in the default? Walking through the code, looks like we'll add it twice?

Reply:
If you meant to put this here for tests, there's a `testConfiguration` you can add it to.

Edit: nevermind, i see you already did that.

Reply:
In vim, is there a way to configure this in `.vimrc`? If so, we should match similar naming.

Reply:
> Should I rename it to just digraphs instead of customDigraphs
Yes, I think that would be best.